service/src/main/java/uk/gov/hmcts/reform/fpl/model/DocumentsSentToParty.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.utils.ElementUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+
+@Data
+@Builder(toBuilder = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DocumentsSentToParty {
+    private String partyName;
```

Comment:
Is it intentional that `partyName` isn't final whilst the list is?

Reply:
probably not, picked from another branch, fixed